src/erasure-code/jerasure/ErasureCodePluginJerasure.cc

```diff
@@ -54,11 +54,10 @@ int ErasureCodePluginJerasure::factory(const std::string& directory,
     } else if (t == "liber8tion") {
       interface = new ErasureCodeJerasureLiber8tion();
     } else {
-      derr << "technique=" << t << " is not a valid coding technique. "
+      *ss << "technique=" << t << " is not a valid coding technique. "
```

Comment:
@tchaikov . Should we check ss != nullptr?

Reply:
no, i don't think so. please see `ErasureCode::sanity_check_k()`. FWIW, i think the reason why we have `ss` as a pointer here, is merely that `ss` is not a `const` here, and we have a tradition to pass non-const parameter as pointer.